wandb/sdk/lib/filesystem.py

```diff
@@ -240,6 +242,97 @@ def safe_copy(source_path: StrPath, target_path: StrPath) -> StrPath:
     return target_path
 
 
+def _reflink_linux(target: StrPath, new_link: StrPath) -> None:
+    """Create a reflink to `target` at `new_link` on Linux."""
+    import fcntl
+
+    FICLONE = 0x40049409  # magic number from <linux/fs.h>  # noqa: N806
+    with open(target, "rb") as t_f, open(new_link, "wb+") as l_f:
+        fcntl.ioctl(l_f.fileno(), FICLONE, t_f.fileno())
+
+
+def _reflink_macos(target: StrPath, new_link: StrPath) -> None:
+    try:
+        clib = ctypes.CDLL("libc.dylib", use_errno=True)
+    except OSError as e:
+        if e.errno != errno.ENOENT:
+            raise
```

Comment:
I guess you wanted to ignore the error when `libc.dylib` doesn't exist. For me `errno` is not set in this case:
```
>>> try:
...   ctypes.CDLL("dscsdcsd", use_errno=True)
... except OSError as e:
...   print(e.errno)
...
None
```

Reply:
Oops, you're right.